assets/js/src/newsletters/campaign_stats/page.jsx

```diff
@@ -73,8 +73,8 @@ class CampaignStatsPage extends React.Component {
 
     MailPoet.Ajax.post({
       api_version: window.mailpoet_api_version,
-      endpoint: 'stats',
-      action: 'get',
+      endpoint: window.mailpoet_premium_active ? 'stats' : 'newsletters',
+      action: window.mailpoet_premium_active ? 'get' : 'getWithStats',
```

Comment:
What's the difference between those two? Do they return different data?

Reply:
`stats` API is available only in the premium plugin, so in free I had to call `newsletters` API, which returns basic stats like opens, clicks and unsubscribes